other/db/data/tables.sql

```diff
@@ -241,18 +241,20 @@ CREATE FUNCTION search_bgpupdates_by_as_path_and_hijack_key(key text, as_paths B
 $$ LANGUAGE sql STABLE;
 
 CREATE TABLE IF NOT EXISTS dataplane_msms (
-    key VARCHAR ( 32 ) NOT NULL,
+    key VARCHAR (32) NOT NULL,
```

Comment:
please also apply the following changes w.r.t. msms table and view on `testing/detection/db/data/tables.sql`

Reply:
btw do not worry about any travis-ci errors they relate to this being a forked PR, I will merge the PR securely after you also apply the change in the test tables.sql

Reply:
@vkotronis Οκ! I fixed it.